docs/components/DocumentationPage.js

```diff
@@ -81,13 +81,10 @@ export default class DocumentationPage extends React.Component {
 
   _handleSetVersion = version => {
     this._version = version;
-    let newPath = '/versions/' + version;
-
-    // TODO: Find what's stripping trailing slashes from these
-    if (version.startsWith('v')) {
-      newPath += '/';
-    }
+    const newPath = Utilities.replaceVersionInUrl(this.props.url.pathname, version);
 
+    // note: we can do this without validating if the page exists or not.
```

Comment:
Are you saying here that b/c of the way the docs redirect logic is already set-up, we can just append on a "/" and call it a day (i.e., w/o validating the link here b/c it will validate it later?)

Reply:
Almost, I'm saying that we can just replace the version in the URL "and hope for the best". We don't have to check if the page exists, because we have an error-flow in case it isn't (which still need the banner from the comments earlier).